txnsync/bitmask_test.go

```diff
@@ -168,3 +172,47 @@ func trimIterateHelper(t *testing.T, setBits []int) {
 		}
 	}
 }
+
+func TestFuzzBitmask(t *testing.T) {
+	randSeed := uint64(0)
+	rand := func() byte {
+		bytes := [16]byte{}
+		l := binary.PutUvarint(bytes[:], randSeed)
+		h := crypto.Hash(bytes[:l])
+		randSeed = 0
+		for i := 0; i < 8; i++ {
+			randSeed += uint64(h[i]) << (i * 8)
+		}
+		return byte(h[0])
+	}
+	for iterationsCount := 0; iterationsCount < 1000; iterationsCount++ {
+		bitmaskType := rand() % 4
+		blen := int(rand()%33) + 1
+		var b bitmask
+		b = make([]byte, blen)
+		b[0] = byte(bitmaskType)
+		for i := 1; i < blen; i++ {
+			b[i] = rand()
+		}
+		entries := int(rand())
+		maxIndex := int(rand())
+		lastEntryIndex := -1
+		b.iterate(entries, maxIndex, func(i, j int) error {
```

Comment:
Testing the return of this is important. 
Code is getting covered because of TestFuzzBitmask, but the covered code is not verified by the test.